buildpack.md

```diff
@@ -13,36 +13,27 @@ This is accomplished in four phases:
 
 The `ENTRYPOINT` of the OCI image contains logic implemented by the lifecycle that executes during the **Launch** phase.
 
-Additionally, a lifecycle can use buildpacks to create a containerized environment for developing or testing application source code.
-
-This is accomplished in two phases:
-
-1. **Detection,** where an optimal selection of compatible buildpacks is chosen.
-2. **Development,** where the lifecycle uses those buildpacks to create a containerized development environment.
-
 ## Table of Contents
 
 1. [Buildpack Interface](#buildpack-interface)
```

Comment:
We should take advantage of Markdown's ability to type 1., 1., 1., 1. and have it create and `<ol>` that chooses numbers.

Reply:
I will happily PR in that change separately after this merges, but I want to keep this PR scoped to the topic at hand